samples/sensor/bme280/src/main.c

```diff
@@ -13,6 +13,11 @@ void main(void)
 {
 	struct device *dev = device_get_binding("BME280");
 
+	if (dev == NULL) {
+		printf("Could not get BME280 device\n");
```

Comment:
Use printk here too

Reply:
The code already uses printf everywhere else.  I think its a different issue to convert this sample to use printk.